cucumber/src/main/java/org/chodavarapu/datamill/cucumber/PlaceholderResolver.java

```diff
@@ -19,6 +21,7 @@
     private static final Pattern PLACEHOLDER_PATTERN = Pattern.compile("\\{([^\\}]+)\\}");
     private static final String RANDOM_ALPHANUMERIC_PLACEHOLDER_PREFIX = "randomAlphanumeric";
     private static final String HASHED_PLACEHOLDER_PREFIX = "blowfish:";
+    private static final String DATE_FORMATTER_PLACEHOLDER_PREFIX = "dateFormatter:";
```

Comment:
I think this should be "now:" or "currentTime:" instead of "dateFormatter:" so it's clear it's the current time being formatted